### DISCUSSION TITLE
File format for android and ios

### DISCUSSION THREAD

**Discussion by Arumugamsix** [google/model-viewer#4130] [Q&A]
Can we move the object to the top and bottom in AR mode?

Can we same file format for android and iOS?

**Comment by elalish** [comment]
If you mean place the object on the ceiling, I believe Quick Look mode does that on iOS (though it turns the model upside-down). We don't currently support it on Android.

Yes, you can use only a GLB and we will auto-generate a USDZ on the fly for iOS: https://modelviewer.dev/examples/augmentedreality/#webXR
